src/assets/scss/main.scss

```diff
@@ -435,8 +435,10 @@
 				font-size: 30px;
 			}
 			.icon-weather {
-				width: 150px;
-				height: 150px;
+				img{
```

Comment:
2 indentations produces more compact and readable code, fix this file. 
Also, add one indentation after selector name `.temp {`.